net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -513,22 +557,83 @@ function quagga_ospfd_plugin_carp($pluginparams) {
 		return null;
 	}
 	/* If there is no properly configured CARP status check IP, then stop */
-	if (!isset($ospfd_conf['carpstatusvid']) || $ospfd_conf['carpstatusvid'] == "none") {
-		return null;
-	}
-	list($vhid, $iface) = explode("@", trim($pluginparams['interface']));
-	$friendly = convert_real_interface_to_friendly_interface_name($iface);
-	$vip = get_configured_vip($ospfd_conf['carpstatusvid']);
-	if ($vip['vhid'] != $vhid || $vip['interface'] != $friendly) {
-		return null;
-	}
-
-	/* Start or stop the service as needed based on the CARP transition. */
-	if ($pluginparams['event'] == "rc.carpmaster") {
-		start_service("Quagga OSPFd");
-	} elseif ($pluginparams['event'] == "rc.carpbackup") {
-		stop_service("Quagga OSPFd");
-	}
+    if (!isset($ospfd_conf['carpmode']) || $ospfd_conf['carpmode'] == "none") {
```

Comment:
Leading spaces should be tabs, affects everything in this whole section of the diff

Reply:
all whitespace issues are fixed